src/app/state/mod.rs

```diff
@@ -46,7 +45,6 @@ impl Default for Tabs {
 pub struct AppState<'app> {
     pub config: AppConfig,
     pub should_quit: bool,
-    pub data_dir: PathBuf,
```

Comment:
This field is not used, and I was trying to reduce the overhead of running tests (e.g. not open files if it isn't necessary)

Reply:
I just double checked and that field was from the Polygon TUI I was working on and specific to that app - so good to remove